libraries/AC_Avoidance/AP_OABendyRuler.h

```diff
@@ -45,10 +49,16 @@ class AP_OABendyRuler {
     // on success returns true and updates margin
     bool calc_margin_from_object_database(const Location &start, const Location &end, float &margin);
 
-    // configuration parameters
-    float _lookahead;               // object avoidance will look this many meters ahead of vehicle
+    // OA common parameters
     float _margin_max;              // object avoidance will ignore objects more than this many meters from vehicle
+    
+    // BendyRuler parameters
+    AP_Float _lookahead;            // object avoidance will look this many meters ahead of vehicle
+    AP_Float _bendy_ratio;          // object avoidance will avoid major directional change if change in margin ratio is less than this param
+    AP_Int16 _bendy_angle;          // object avoidance will try avoding change in direction over this much angle
 
     // internal variables used by background thread
     float _current_lookahead;       // distance (in meters) ahead of the vehicle we are looking for obstacles
+    float _stored_bearing;          // stored bearing in degrees 
```

Comment:
the "_stored_bearing" name is ok but you could also call it "_bearing_last" or "_bearing_prev".  I'm just thinking that we called another variables "_destination_prev" to indicate the same sort of thing so might as well make the naming convention consistent in this file.